src/app/find/locationDetails/CategoryCard.jsx

```diff
@@ -1,150 +1,40 @@
-import React, { Fragment } from 'react';
-import moment from 'moment';
-import { DAYS, OCCASIONS } from '../../../Constants';
+import React, { Component } from 'react';
 import { getCategoryIcon } from '../../../services/iconography';
 import Icon from '../../../components/icon';
-import PhoneLink from '../../../components/phoneLink';
-import InfoItem from './InfoItem';
-import ServiceRestrictions from './ServiceRestrictions';
-import ServiceOfferings from './ServiceOfferings';
 import './locationDetails.css';
+import ShelterService from './ShelterService';
 
-const renderSchedule = (schedule) => {
-  if (schedule.length === 7 &&
-    schedule.every(day => day.opens_at === '00:00:00' && day.closes_at === '23:59:00')) {
-    return 'Open 24/7';
+// this card contains a category (e.g. Food) and displays
+// all services that a shelter provides within that category
+class CategoryCard extends Component {
+  state = {
```

Comment:
Looks like this state and the `handleCollapse` function are no longer used, so they can be removed and this component changed back into a functional component.